text/0999-make-hash-built-in.md

```diff
@@ -30,55 +27,42 @@ project-link: Leave as is
 suite: Leave as is
 -->
 
-# <RFC title>
+# Make `(hash)` a built in helper 
 
 ## Summary
 
-> One paragraph explanation of the feature.
+Today, when using gjs/gts/`<template>`, in order to bind event listeners, folks _must import_ the `(hash)` helper.
+Because creating arrays is fairly commonplace, this is a grating annoyance for developers.
+
+This RFC proposes that `(hash)` be built in to `glimmer-vm` and not require importing.
 
 ## Motivation
 
-> Why are we doing this? What use cases does it support? What is the expected
-outcome?
+Arrays and Objects are not only very common to create, they are essential tools when yielding data out of components.
 
-## Detailed design
+There is alternate motivation to implement _literals_ for arrays and objects, but that is a bigger can of worms for another time.
 
-> This is the bulk of the RFC.
+## Detailed design
 
-> Explain the design in enough detail for somebody
-familiar with the framework to understand, and for somebody familiar with the
-implementation to implement. This should get into specifics and corner-cases,
-and include examples of how the feature is used. Any new terminology should be
-defined here.
+This change would affect strict-mode only. This is so that today's existing code that imports `hash` from `@ember/helper` will still work due to how values define locally in scope override globals.
```

Comment:
```suggestion
This change would affect strict-mode only. This is so that today's existing code that imports `hash` from `@ember/helper` will still work due to how values defined locally in scope override globals.
```